service-front/module/Application/view/application/authenticated/lpa/correspondent/index.twig

```diff
@@ -136,13 +136,16 @@
 
         <div class="form-group {{ topLevelError ? 'form-group-error'}}">
 
-            <fieldset id="correspondence">
-
+            <fieldset id="{{ correspondence.getAttribute('name') }}">
                 <legend class="visually-hidden">Contact options</legend>
                 <h3 class="heading-medium flush--top">How would the person above prefer to be contacted?</h3>
 
                 <p>Select how the correspondent would like to be contacted. You can choose more than one option.</p>
 
+                {% if topLevelError %}

```

Comment:
This is required as if there is an error on either/both of phone/email elements they will also display the error message here. Which is not required & would cause confusion.

Reply:
Have to be careful here: the ID should be on the top checkbox in the set, rather than on the fieldset. This is so that the message shown in the error summary at the top of the page links to the checkbox, not the surrounding fieldset; see https://opgtransform.atlassian.net/browse/LPAL-248